### PR TITLE
fix: crashed after local pairing

### PR DESCRIPTION
**Pull request by qfrank**
this PR should fixed [issue 4](https://github.com/status-im/status-mobile/pull/14763#issuecomment-1385506833)

### CODE REVIEW

**Review by siddarthkay — APPROVED** [review]
looks okay, please fix dangling parentheses on line 191 :)

File: src/status_im/utils/keychain/core.cljs [hunk 1]
```diff
@@ -173,18 +173,22 @@
...
+      (str key-uid "-auth")
+      key-uid
+      method
+      #(when-not %
+         (log/error
+          (str "Error while saving auth method."
+               " "
+               "The app will continue to work normally, "
+               "but you will have to login again next time you launch it.")))))
+ ))
```
Comment (siddarthkay) [inline src/status_im/utils/keychain/core.cljs]: @qfrank : this is what i meant by dangling parenthesis :)

**Comment by qfrank** [comment]
> looks okay, please fix dangling parentheses on line 191 :)

sure, thanks for review🫶

**Comment by qfrank** [comment]
@pavloburykh 😊 , pls try this fix

**Comment by pavloburykh** [comment]
> @pavloburykh 😊 , pls try this fix

thanx, I will check and provide an update. Just one question: why are we fixing this issue in separate PR? Shouldn't we fix in original PR, where issue has been localized https://github.com/status-im/status-mobile/pull/14763 ?

**Comment by qfrank** [comment]
> > @pavloburykh 😊 , pls try this fix
> 
> thanx, I will check and provide an update. Just one question: why are we fixing this issue in separate PR? Shouldn't we fix in original PR, where issue has been localized #14763 ?

emm.. that PR was initially created only for testing local pairing, not meant to be merge, and i want a review for this fix.. (it makes my commit history more beatiful, don't take this seriously :) )

**Comment by pavloburykh** [comment]
@qfrank thanx for your work. 

I have checked the fix: good news is that crash is not happening any more, user is being redirected into account. Though, I should admit that scanning QR takes pretty long time before user is finally redirected into account, but I guess this can be improved in future. 

Some questions and issues that still exist

### ISSUE/QUESTION 1 Once QR code is generated it can be scanned for 1 time

Steps:

1. Generate sync QR code by User 1
2. Install app by User 2 
3. Scan QR code by User 2 during onboarding (wait for redirection into account)
4. Do not close QR code by User 1
5. Reinstall app by User 2
6. Try to scan the same QR code again
7. Observe the result

**Actual result:** User 2 will not be redirected into account after scanning the old QR code. 

**Expected result:** is it expected? 

Below you can see a video recorded on IOS. 

[logs (13).zip](https://github.com/status-im/status-mobile/files/10544282/logs.13.zip)

https://user-images.githubusercontent.com/97245802/215706754-d344eba3-34d5-41a9-94ba-fbcd68f509d0.mp4


Other issues that still exist:

### ISSUE 2 Add contact button doesn't respond after redirection into account (until app relaunch)

### ISSUE 3 Unable to open Scan QR from login screen after redirection into account (until app relaunch)

Below video is demonstrating ISSUE 1 and ISSUE 2 

https://user-images.githubusercontent.com/97245802/215712807-d08e74e5-74b6-477f-85c8-3bf7bf7ae390.mp4

**Comment by qfrank** [comment]
Hi @pavloburykh , for issue 1, yes, it is expected, but i'm not sure why we need this limit, we need explain from @Samyoul or maybe @cammellos know something.. for issue 2-3, i think they're the same thing, there is a [PR](https://github.com/status-im/status-mobile/pull/14936) for this, let's merge this PR first, thank you!

**Pull request merged**